lib/grpc/endpoint.ex

```diff
@@ -94,12 +94,29 @@ defmodule GRPC.Endpoint do
     server_interceptors(tail, acc)
   end
 
-  defp parse_servers(servers) do
+  defp parse_servers(servers, false) do
+    servers = [{GRPC.Reflection.Service, %{}}] ++ servers
+
     servers
     |> Enum.map(fn {server, _} -> server end)
     |> List.flatten()
   end
 
+  defp parse_servers(servers, true) do
+    servers
+    |> Enum.map(fn {server, _} -> server end)
+    |> List.flatten()
+  end
+
+  defp has_reflection_disabled?() do
```

Comment:
IMO, I think it's more common/easy to read if you check for the feature to be enabled. 
Also, the convention in elixir is to have the bolean funcs suffixed with `?` and guards using `is_something`. Thoughts?
```suggestion
  defp reflection_enabled?() do
```